.NET/Microsoft.Recognizers.Text.Sequence/Chinese/Extractors/ChineseURLExtractorConfiguration.cs

```diff
@@ -6,7 +6,6 @@ namespace Microsoft.Recognizers.Text.Sequence.Chinese
 {
     public class ChineseURLExtractorConfiguration : URLConfiguration
     {
-
```

Comment:
Nit: Please don't remove blank lines unless there's a reason for it. :)

Reply:
> Nit: Please don't remove blank lines unless there's a reason for it. :)

This blank line is unnecessary. Remove it can keep format same with Config file.